roles/ceph-docker-common/tasks/pre_requisites/remove_redhat_ceph_packages.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: list ceph redhat packages
+  shell: |
+    rpm -qa | grep -E "ceph|librbd|librados"
```

Comment:
seems too generic, we'd end up removing a lot of packages for cascading dependencies

we probably want to remove only the leaf packages here, ceph-mon, ceph-osd, -mds, -test, -radosgw, -mgr

Reply:
:+1: purging packages like this needs a lot of testing